lib/typescript/httpclient/endpoints/listConversations.ts

```diff
@@ -5,12 +5,12 @@ import {ConversationPayload} from '../payload/ConversationPayload';
 import {MessagePayload} from '../payload/MessagePayload';
 import {PaginatedPayload} from '../payload/PaginatedPayload';
 
-const messageMapper = (payload: MessagePayload): Message => {
+export const messageMapper = (payload: MessagePayload): Message => {
```

Comment:
the mappers are private "by choice" as there's no need to export this (from the perspective of the api lib I mean). I also can't find where we're using this. But I imagine we may need to extract it from this endpoint and use it in one than one place. When I discussed this with @AudreyKj we thought to wait until it was really needed (now it seems so) before introducing one more folder (`httpclient/mappers` ?). But the point of not making those importable from the lib stays